eras/dijkstra/impl/cddl-files/dijkstra.cddl

```diff
@@ -419,7 +419,7 @@ pool_metadata = [url, bytes]
 
 url = text .size (0 .. 128)
 
-epoch_no = uint .size 8
+epoch = uint .size 8
```

Comment:
This definition can also travel into cardano-ledger-core and reused everywhere, since prior eras also wrongfully used unsized version `uint`